app/src/main/java/com/x8bit/bitwarden/data/platform/manager/model/AppCreationState.kt

```diff
@@ -3,14 +3,15 @@ package com.x8bit.bitwarden.data.platform.manager.model
 /**
  * Represents the creation state of the app.
  */
-enum class AppCreationState {
+sealed class AppCreationState {
     /**
      * Denotes that the app is currently created.
+     * @param createdForAutofill Whether the app was created for autofill.
      */
-    CREATED,
+    data class Created(val createdForAutofill: Boolean) : AppCreationState()
```

Comment:
Can we make this `isCreatedForAutofill` or `wasCreatedForAutofill`? I think that might fit boolean patterns in the app a little better. Or maybe even `isAutofill`?